osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/utils/JavaTimeHelpers.java

```diff
@@ -34,4 +38,23 @@ public static int getOffsetForZonedDateTimeInMillis(final ZonedDateTime dateTime
   public static boolean isDayLightSavingsActive(final ZonedDateTime dateTime) {
     return dateTime.getZone().getRules().isDaylightSavings(dateTime.toInstant());
   }
+
+  public static ZonedDateTime shiftZoneToUTC(final ZonedDateTime dateTime) {
+    final int offset = dateTime.getZone().getRules().getOffset(Instant.now()).getTotalSeconds();
+    return dateTime.plusSeconds(offset).withZoneSameInstant(ZoneId.of("UTC"));
+  }
+
+  public static ZonedDateTime parseToZonedDateTime(final String date) {
```

Comment:
Ik vind dit wel een beetje enge functie. waarom moeten we gokken wat voor datum formaat we krijgen?

Reply:
Als het nodig is kunnen we ook meerdere formaten mee geven aan de ZonedDateTime in plaats van proberen met andere classes.

Reply:
Zoals besproken wat Java  doc toegevoegd en hem gemarkeerd als deprecated

Reply:
Ik zie de JavaDoc nog niet

Reply:
Dat is wel apart omdat die in mijn codebase er wel bijstaat